nailgun/entities.py

```diff
@@ -1042,6 +1042,7 @@ def __init__(self, server_config=None, **kwargs):
             'attach_stdin': entity_fields.BooleanField(),
             'attach_stdout': entity_fields.BooleanField(),
             'command': entity_fields.StringField(
+                default='top',
                 required=True,
                 str_type='latin1',
```

Comment:
btw i have questions to this line - is it supposed to be `latin1`? server is returning 500 error each time when random latin1 string is sent as a command, and basically all our tests are defining custom command string (probably because of that).